inc/ustring.bi

```diff
@@ -0,0 +1,681 @@
+
```

Comment:
If this PR is for whitespace, then I do not expect to see a new file inc/ustring.bi added.  ustring.bi is also added in PR #155 and the 2 versions are different.  Maybe this was added by mistake?